js/game.js

```diff
@@ -1821,12 +1830,22 @@ export class Level extends LevelInterface {
         }
     }
 
+    _push_pending_undo(thunk) {
```

Comment:
I tried inlining this helper but that didn't seem to make any difference to performance, so i kept it.